_gsocproposals/2018/proposal_TMVAGANs.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Generative Adversarial Networks for Particle Physics Applications
+layout: gsoc_proposal
+project: TMVA
+year: 2018
+organization:
+  - UdeA
+  - Florida
+  - CERN
+  - KIT
+  - OProject
+---
+
+# Description
+
+The use of Generative Adversarial Networks (GANs) is of particular interest for potential applications for particle physics in the area of fast detector simulations and reducing the systematic uncertainty related to applying machine learning algorithms. GANs are sets of networks competing with each other. For example a generator network produces simulated samples based on a training set, competing with a classification network that attempts to classify events. The use of GANs is of particular interest for potential applications focused on fast and accurate event simulation as well as other applications aimed at reducing the dependence of the classifier on specific parameters. 
```

Comment:
"for potential applications" -> "because of potential applications" (too many "for"s).
Missing comma after "For example".